src/App/Action/Account/Meetup/CancelCheckInAction.php

```diff
@@ -56,6 +56,6 @@ public function __invoke(Request $request, Response $response, callable $next =
             $meetup->cancelCheckIn($user);
         });
 
-        return new RedirectResponse($this->urlHelper->generate('account-meetup-view', ['uuid' => $meetup->getId()]));
+        return new RedirectResponse($this->urlHelper->generate('account-meetup-view', ['uuid' => $meetup->getId()], [], 'attendees'));
```

Comment:
Just the line length is too long now :smile:

Reply:
😱